libs/utils/include/utils/compiler.h

```diff
@@ -115,7 +115,13 @@
 #define UTILS_UNUSED_IN_RELEASE
 #endif
 
-#define UTILS_RESTRICT __restrict__
+#if defined _MSC_VER && _MSC_VER >= 1900
```

Comment:
```suggestion
#if defined(_MSC_VER) && _MSC_VER >= 1900
```

Consistent parenthesis use